src/imports/background/state-manager.test.ts

```diff
@@ -111,6 +111,21 @@ const runSuite = (DATA: TestData) => async () => {
         await testEstimateCounts()
     })
 
+    test('counts calcs should be consistent', async () => {
+        state.dirtyEsts()
```

Comment:
Could dirtyEsts() get a more descriptive name? Don't know what it does, but I'm guessing something like setEstimationMode('dirty') or resetEstimations({mode: 'dirty'}) ?

Reply:
It just dirties the cache state so that the next time estimates are requested it will get a cache miss and will be forced to go through the complex steps of recalculation rather than reading from cache. What do you think about `dirtyEstsCache()`?